[2025-10-21T20:07:47.509+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-10-21T20:07:47.705+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: airbnb_etl_pipeline.transform_airbnb_listings manual__2025-10-21T19:57:05.716903+00:00 [queued]>
[2025-10-21T20:07:47.727+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: airbnb_etl_pipeline.transform_airbnb_listings manual__2025-10-21T19:57:05.716903+00:00 [queued]>
[2025-10-21T20:07:47.729+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2025-10-21T20:07:47.753+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): transform_airbnb_listings> on 2025-10-21 19:57:05.716903+00:00
[2025-10-21T20:07:47.762+0000] {standard_task_runner.py:63} INFO - Started process 338 to run task
[2025-10-21T20:07:47.773+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'airbnb_etl_pipeline', 'transform_airbnb_listings', 'manual__2025-10-21T19:57:05.716903+00:00', '--job-id', '30', '--raw', '--subdir', 'DAGS_FOLDER/airbnb_etl_dag.py', '--cfg-path', '/tmp/tmpeqmsrhrv']
[2025-10-21T20:07:47.780+0000] {standard_task_runner.py:91} INFO - Job 30: Subtask transform_airbnb_listings
[2025-10-21T20:07:47.967+0000] {task_command.py:426} INFO - Running <TaskInstance: airbnb_etl_pipeline.transform_airbnb_listings manual__2025-10-21T19:57:05.716903+00:00 [running]> on host b58697e26e05
[2025-10-21T20:07:48.246+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='airbnb_etl_pipeline' AIRFLOW_CTX_TASK_ID='transform_airbnb_listings' AIRFLOW_CTX_EXECUTION_DATE='2025-10-21T19:57:05.716903+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-10-21T19:57:05.716903+00:00'
[2025-10-21T20:07:48.247+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-10-21T20:07:48.279+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-10-21T20:07:48.282+0000] {taskinstance.py:2905} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/baseoperator.py", line 401, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/operators/python.py", line 235, in execute
    return_value = self.execute_callable()
                   ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/operators/python.py", line 252, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/scripts/etl_script.py", line 575, in transform1
    airdna_data = pd.json_normalize(all_listings)
                                    ^^^^^^^^^^^^
NameError: name 'all_listings' is not defined
[2025-10-21T20:07:48.325+0000] {taskinstance.py:1206} INFO - Marking task as UP_FOR_RETRY. dag_id=airbnb_etl_pipeline, task_id=transform_airbnb_listings, run_id=manual__2025-10-21T19:57:05.716903+00:00, execution_date=20251021T195705, start_date=20251021T200747, end_date=20251021T200748
[2025-10-21T20:07:48.365+0000] {standard_task_runner.py:110} ERROR - Failed to execute job 30 for task transform_airbnb_listings (name 'all_listings' is not defined; 338)
[2025-10-21T20:07:48.434+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 1
[2025-10-21T20:07:48.490+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-10-21T20:07:48.494+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
